src/qtools/rad.cpp

```diff
@@ -358,8 +357,7 @@ void CalcFaceExtents(Bsp* bsp, lightinfo_t* l)
 
 	s = l->face;
 
-	mins[0] = mins[1] = FLT_MIN;
-	maxs[0] = maxs[1] = FLT_MAX;
+	bool initminmax = false;
 

```

Comment:
Only change in this commit should be reverting this change. It used to be:
```
mins[0] = mins[1] = FLT_MAX;
maxs[0] = maxs[1] = FLT_MIN;
```
That should fix the bad surface extents (min/max were swapped).

Reply:
No, You can test, FLT_MAX/FLT_MIN broke all maps ))

Reply:
(Cause random lightmaps and if select any face, it write 'surface extents')

Reply:
ok I tried without FLT_MIN/MAX and still broken. Maybe revert all changes in this file? The initminmax stuff might be less efficient, and this code needs to be as fast as possible.

Reply:
@wootguy previous values :

**mins[0] = mins[1] = 999999;
maxs[0] = maxs[1] = -99999;**

and 

**mins[0] = mins[1] = 99999999;
maxs[0] = maxs[1] = -99999999;**

It works. But if replace it to FLT_MAX FLT_MIN, it cause problems in all maps.

**If just initialize mins/maxs with first value from list, it works in all maps.**

Reply:
I checked this method works fine without FLT_MIN / FLT_MAX;

Reply:
wow that is strange. I just noticed there were multiple places were FLT_MIN/MAX were used here. Reverting both of those changes fixed it.

Reply:
Ok, can you update the request to only revert these lines then?
https://github.com/wootguy/bspguy/blob/master/src/qtools/rad.cpp#L305
https://github.com/wootguy/bspguy/blob/master/src/qtools/rad.cpp#L362

Reply:
I can just increase code speed ? https://github.com/wootguy/bspguy/pull/34/commits/d92d5155f06e8b18d211961a53ddf9d3049f237f

Reply:
I just don't understand why there is an error with FLT_MIN / MAX, but this totally fixes it))

Reply:
How I can test map load speed with these and old code ?

Reply:
merging maps is a good way to benchmark, which does a lot of lightmap calculations.